src/api/app/models/relationship.rb

```diff
@@ -126,6 +128,13 @@ def check_global_role
     errors.add(:base,
                "global role #{role.title} is not allowed.")
   end
+
+  # NOTE: Adding a normal validation, the error doesn't reach the view due to
```

Comment:
So this explains why we cannot include `AllowedUserValidator` in this model? Just making sure I understand correctly.